operations/network/management_network/bican_support_matrix.md

```diff
@@ -0,0 +1,104 @@
+
+* Perform administrative tasks on the system.
+* Run jobs and move job data to and from the system.
+* Access site resources like DNS and LDAP from the system.
+
+## Feature access matrix
+
+For CSM 1.2, the notion of the CAN has been expanded to meet customer requests for increased flexibility and policy control.
+
+| System Resource | Traffic to and from System | Management Network or CAN | High Speed Network CHN | Management Network CMN       |
+| --------------- | :------------------------: | :-----------------------: | :--------------------: |:-----------------------: |
+| System Cloud Resources (APIs) | Ingress                    | Jobs-related APIs         | Jobs-related APIs      | Administrative APIs          |
+| Application Node Servers (UAI, UAN, re-purposed CN) | Ingress                    | Allowed                   | Allowed                | Not Allowed                  |
+| Non-Compute Node (NCN) Servers | Ingress                    | Not Allowed               | Not Allowed            | Allowed                      |
+| System Access to External/Site (LDAP, DNS) | Egress                     | Allowed                   | Allowed                | Not Allowed                  |
+
+* Selection of user access for job control and data movement over the Shasta Management Network (CAN) or the High Speed Network (CHN) is made during system installation or upgrade.
+
+* Creation of the Customer Management Network (CMN) during installation or upgrade is mandatory.
+
+## Network overview
+
+![tds can overview](img/tds_can_overview.png)
+
+### Internal networks
+
+* Node Management Network (NMN) \- Provides the internal control plane for systems management and jobs control.
+* Hardware Management Network (HMN) \- Provides internal access to system baseboard management controllers (BMC/iLO) and other lower-level hardware access.
+
+### External and edge networks
+
+* Customer Management Network (CMN) \- Provides customer access from the site to the system for administrators.
+* Customer Access Network (CAN) or Customer High Speed Network (CHN) provide:
+  * Customer access from the site to the system for job control and jobs data movement.
+  * Access from the system to the site for network services like DNS, LDAP, etc.
+
+## Supported configurations
+
+### Option A: CMN + CAN (Management Network only - Layer 2 separation)
+
+![cmn plus can](img/cmn_plus_can.png)
+
+### Option B: CMN + CHN (Administration over Management Network, User Access over High Speed Network)
+
+![cmn plus chn](img/cmn_plus_chn.png)
+
+Note: During installation, the High Speed Network is not configured until relatively late in the process.
+Installation generally requires site access for deployment artifacts, site DNS, etc.
+In order to achieve this, the Management Network CAN is used during the installation process for system traffic egress until the High Speed Network is available.
```

Comment:
`Management Network CAN` Might want to change to CMN, calling it the CAN might be confusing.

Reply:
updated to mention CMN instead